### PR TITLE
OADP-844: add to known issues - generated route can have incorrect hostnamect ho…

### PR DESCRIPTION
**Pull request by anarnold97**
* [OADP-844](https://issues.redhat.com/browse/OADP-844)

Version(s):

Enterprise 4.11 → branch/enterprise-4.11
Enterprise 4.12 → branch/enterprise-4.12
Enterprise 4.13 → branch/enterprise-4.13
Enterprise 4.14 → branch/enterprise-4.14
Enterprise 4.15 → branch/enterprise-4.15

Issue:

https://issues.redhat.com/browse/OADP-844

## Link to docs preview:

* [OADP 1.2.0 - Known issues](https://65626--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/oadp-release-notes#known-issues-1-2-0_oadp-release-notes)

## QE review:
- [ X] [[QE has approved this change](https://github.com/openshift/openshift-docs/pull/65626#pullrequestreview-1661213263).

Additional information:

### CODE REVIEW

**Review by stillalearner — APPROVED** [review]
lgtm

File: modules/oadp-release-notes-1-2-0.adoc [hunk 1]
```diff
@@ -34,5 +34,14 @@ For a complete list of all issues resolved in this release, see the list of link
 [id="known-issues-1-2-0_{context}"]
 == Known issues
 
-This release does not have any known issues.
+The following issue has been highlighted as known issues in the release of OADP 1.2.0:
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/oadp-release-notes-1-2-0.adoc:37]: https://redhat-documentation.github.io/supplementary-style-guide/#shortdesc

> Use active voice and present tense.

**Review by xenolinux — COMMENTED** [review]
I left a few suggestions/questions but otherwise looks good!

I am new on the peer review squad. Please wait until @abhatt-rh gives my review the go-ahead before implementing any feedback. Thanks!

**Comment by abhatt-rh** [comment]
> I left a few suggestions/questions but otherwise looks good!
> 
> I am new on the peer review squad. Please wait until @abhatt-rh gives my review the go-ahead before implementing any feedback. Thanks!

Hi @anarnold97 I am not sure if I understand how and when hostnames are affected by different routes (generated by OpenShift and generated by a user). I noticed that this content is copied verbatim from a comment in the linked Jira issue.
May I request you to please consider revisiting and editing the content? You may consider @xenolinux 's suggestions for the use of active voice, and the guidance on the term "user".

I am removing the peer-review-done label for now. Please add the necessary label to re-queue. Thanks!

**Comment by anarnold97** [comment]
> > I left a few suggestions/questions but otherwise looks good!
> > I am new on the peer review squad. Please wait until @abhatt-rh gives my review the go-ahead before implementing any feedback. Thanks!
> 
> Hi @anarnold97 I am not sure if I understand how and when hostnames are affected by different routes (generated by OpenShift and generated by a user). I noticed that this content is copied verbatim from a comment in the linked Jira issue. May I request you to please consider revisiting and editing the content? You may consider @xenolinux 's suggestions for the use of active voice, and the guidance on the term "user".
> 
> I am removing the peer-review-done label for now. Please add the necessary label to re-queue. Thanks!

Thank you for your review.

**Review by skrthomas — APPROVED** [review]
/lgtm

File: modules/oadp-release-notes-1-2-0.adoc [hunk 2]
```diff
@@ -34,5 +34,12 @@ For a complete list of all issues resolved in this release, see the list of link
 [id="known-issues-1-2-0_{context}"]
 == Known issues
 
-This release does not have any known issues.
+This release has the following known issues:
 
+.An incorrect hostname can be created when changing a hostname in a generated route.
+
+By default, OpenShift cluster makes sure that the openshift.io/host.generated: 'true' annotation is turned on and fills in the field for both the routes that are generated and those that are not generated.
```
Comment (xenolinux) [inline modules/oadp-release-notes-1-2-0.adoc:41]: @anarnold97  Sorry for this review out of the label process but I got the notification and I happened to check that the backticks are missing here. The annotation should be in backticks: `openshift.io/host.generated: 'true'`

Comment (xenolinux) [reply modules/oadp-release-notes-1-2-0.adoc:41]: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references

**Comment by kalexand-rh** [comment]
I've opened the `enterprise-4.15` branch. Please review your PR and update the applicable versions to also cover 4.15 if you expect that the changes in this PR will be valid in that branch.

**Pull request closed without merging**